tfe/data_source_workspace.go

```diff
@@ -44,6 +44,11 @@ func dataSourceTFEWorkspace() *schema.Resource {
 				Computed: true,
 			},
 
+			"global_remote_state": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
```

Comment:
Needs the associated `remote_state_consumer_ids` schema/setting.

Reply:
added